pkg/functions.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+func ElectionPostChallengeCount(sectors uint64) uint64 {
```

Comment:
Noting that I don't think this belongs here -- feels like it belongs in a different repo altogether. I don't want to block the other work you have depending on this so I guess it will have to be fine for now.